exercises/eliuds-eggs/description.md

```diff
@@ -0,0 +1,50 @@
+# Description
+
+Your friend Eliud inherited a farm from her grandma Tigist. Her granny was an inventor and had a tendency to build things in an overly complicated manner. The chicken coop has a digital display showing an encoded number representing the positions of all eggs that could be picked up.
```

Comment:
Could you use a single line per sentence?

```suggestion
Your friend Eliud inherited a farm from her grandma Tigist.
Her granny was an inventor and had a tendency to build things in an overly complicated manner.
The chicken coop has a digital display showing an encoded number representing the positions of all eggs that could be picked up.
```

Reply:
I should have written the forum post that way. Next time :)

Reply:
now using one line per sentence